tools/bazel_stamp_vars.sh

```diff
@@ -1,7 +1,7 @@
-function onError {
-    echo "Failed to execute: $0"
-    echo ""
-}
+# function onError {
```

Comment:
ignore these changes on your local git cause we don't need these changes in upstream